src/frontend/overlay/src/components/organisms/widgets/Scoreboard.jsx

```diff
@@ -119,7 +119,7 @@ export const ScoreboardRow = ({ teamId, hideTasks, optimismLevel }) => {
             formatPenalty(scoreboardData?.penalty)
         } />}
         {!hideTasks && scoreboardData?.problemResults.map((result, i) =>
-            <ScoreboardTaskResultLabel problemResult={result} key={i}
+            <ScoreboardTaskResultLabel problemResult={result} key={i} problemColor={contestData.problems[i].color}
```

Comment:
Maybe we should create a contest-aware component that will look up the colors from the store, but all we had to do is pass it the problem number?
Something like
```javascript
export const ProblemLabel = ({problemIndex, className}) => {
    const problemData = useSelector((state) => state ...... );
    return <TaskResultLabel problemColor={problemData.color} ..... >
}
```

Reply:
But we are already getting minScore, maxScore, etc. here. Do we want to create a new component for this?

Reply:
Yes. Here we are also getting a lot of other useless data. We should be more specific to make things more pure